Makefile

```diff
@@ -287,7 +287,7 @@ endif
 # be JIT-compiled by the updated driver from the included PTX.
 ifeq ($(USE_CUDA), 1)
 ifeq ($(CUDA_ARCH),)
-	KNOWN_CUDA_ARCHS := 30 35 50 52 60 61 70
+	KNOWN_CUDA_ARCHS := 30 35 50 52 53 60 61 62 70
```

Comment:
could you make the KNOWN_CUDA_ARCHS configurable too?
`ifeq ($(KNOWN_CUDA_ARCHS),) then set the value`.